stingray/modeling/gpmodeling.py

```diff
@@ -37,11 +37,10 @@
     tfp_available = False
 
 
-__all__ = ["get_kernel", "get_mean", "get_prior", 
-           "get_log_likelihood", "GPResult", "get_gp_params"]
+__all__ = ["get_kernel", "get_mean", "get_prior", "get_log_likelihood", "GPResult", "get_gp_params"]
 
 
-def get_kernel(kernel_type, kernel_params):
+def get_kernel(kernel_type, kernel_params, p=1, q=0):
```

Comment:
What range of values of p and q will be used by the user?

Reply:
p and q are user, defined, p will be in the range probably somewhere ~1-10, and similarly q in the range ~0-10, though larger numbers are theoretically possible (but would be very hard to model). An important constraint is that p>=q